docs/getting-started.md

```diff
@@ -166,7 +166,7 @@ Then we can use them in our `lib.rs`:
 ```rust
 pub mod models;
 
-use models::{Blog, Post};
+pub use models::{Blog, Post};
```

Comment:
this is added so that if someone does a build at this stage, they don't get warnings.